recipes/tinygltf/all/test_package/test_package.cpp

```diff
@@ -8,29 +8,13 @@
 #include <string>
 
 int main(int argc, char **argv) {
-  if (argc < 2) {
-    std::cerr << "Need at least one argument\n";
-    return 1;
-  }
 
   tinygltf::Model model;
   tinygltf::TinyGLTF loader;
   std::string err;
   std::string warn;
-  bool ret = loader.LoadBinaryFromFile(&model, &err, &warn, argv[1]);
-
-  if (!warn.empty()) {
-    printf("Warn: %s\n", warn.c_str());
-  }
-
-  if (!err.empty()) {
-    printf("Err: %s\n", err.c_str());
-  }
-
-  if (!ret) {
-    printf("Failed to parse glTF\n");
-    return -1;
-  }
+  bool ret = loader.LoadBinaryFromFile(&model, &err, &warn, "non_existent_file.glb");
+  printf("Test %d\n", ret);
```

Comment:
```suggestion
    loader.LoadBinaryFromFile(&model, &err, &warn, "non_existent_file.glb");
```

I would not check anything as it will always fail.